test/avi/t_insert_mode.clj

```diff
@@ -133,6 +153,16 @@
                     "~" :blue
                     "~" :blue
                     "test.txt" :black :on :white
+                    ""]))
+  (fact "Arrow keys in repeat insert mode"
```

Comment:
I would explicitly call out what's happening here.  Something like "repeat count is ignored when arrows are used during insert".